src/js/components/TaskStatsTable.js

```diff
@@ -0,0 +1,161 @@
+import classNames from 'classnames';
+import moment from 'moment';
+import React from 'react';
+import {Table} from 'reactjs-components';
+
+const taskStatus = ['running','healthy','unhealthy','staged'];
+const keyCaptionMap = {
+  startedAfterLastScaling: 'Started After Last Scaling',
+  withLatestConfig: 'With Latest Config',
+  withOutdatedConfig: 'With Outdated Config',
+  totalSummary: 'Total Summary'
+};
+
+class TaskStatsTable extends React.Component {
+  getTaskStats() {
+    let {taskStats} = this.props;
```

Comment:
I'd like to know why you decided to work with the raw data instead of using the respective structs, as proposed in https://github.com/dcos/dcos-ui/pull/384/files#r66639580